### PR TITLE
YBRO 6 cards

### PR DESCRIPTION
**Pull request by Simisays**
Had to use the inelegant DB Choosecard with Jarsyl, Dark Age Scion since it doesn't target.

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/gixian_recycler.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Gixian Recycler
+ManaCost:2
+Types:Artifact Creature Phyrexian Horror
+PT:3/1
+K:Unearth:1 B
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigConjure | TriggerDescription$ When CARDNAME dies or is put into your graveyard from your hand or library, conjure a card named Gixian Recycler into your graveyard.
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Graveyard | Destination$ Hand,Library | Execute$ TrigConjure | Secondary$ True | TriggerDescription$ When CARDNAME dies or is put into your graveyard from your hand or library, conjure a card named Gixian Recycler into your graveyard.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/gixian_recycler.txt:7]: zones exchanged

File: forge-gui/res/cardsfolder/upcoming/hurkyls_prodigy.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Hurkyl's Prodigy
+ManaCost:2 U
+Types:Creature Human Soldier
+PT:1/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a tapped Powerstone token. (It's an artifact with "{T}: Add {C}. This mana can't be spent to cast a nonartifact spell.")
+SVar:TrigToken:DB$ Token | TokenTapped$ True | TokenScript$ c_a_powerstone
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigUnblockable | TriggerDescription$ Whenever CARDNAME attacks, you may pay {3}. If you go, Hurkyl's Prodigy can't be blocked this turn and it perpetually gets +2/+0.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hurkyls_prodigy.txt:7]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Name:Patchplate Resolute
+ManaCost:3
+Types:Artifact Creature Soldier
+PT:3/3
+K:Unearth:1 W
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigBoon | TriggerDescription$ When CARDNAME enters or leaves the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it."
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Any | Execute$ TrigBoon | Secondary$ True | TriggerDescription$ When CARDNAME enters or leaves the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it."
+SVar:TrigBoon:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ SpellCast | SpellDescription$ You get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.\nUnearth {1}{B}
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt:8]: remove SpellDescription

File: forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+SVar:SpellCast:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | OneOff$ True | ActivationLimit$ 1 | TriggerZones$ Command | Execute$ ReplEffAddCounter | TriggerDescription$ When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.
+SVar:ReplEffAddCounter:DB$ Effect | ReplacementEffects$ ETBAddCounter | RememberObjects$ TriggeredCard
+SVar:ETBAddCounter:Event$ Moved | Origin$ Stack | Destination$ Battlefield | ValidCard$ Card.IsRemembered | ReplaceWith$ ETBAddExtraCounter
+SVar:ETBAddExtraCounter:DB$ PutCounter | ETB$ True | Defined$ ReplacedCard | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ MoveToBattlefield
+SVar:MoveToBattlefield:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Defined$ ReplacedCard | SubAbility$ TrigRemoveSelf
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt:13]: Try using `ReplacementResult$ Updated` instead

File: forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt [hunk 5]
```diff
@@ -5,13 +5,12 @@ PT:3/3
...
+SVar:TrigBoon:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ SpellCast
 SVar:SpellCast:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | OneOff$ True | ActivationLimit$ 1 | TriggerZones$ Command | Execute$ ReplEffAddCounter | TriggerDescription$ When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.
 SVar:ReplEffAddCounter:DB$ Effect | ReplacementEffects$ ETBAddCounter | RememberObjects$ TriggeredCard
 SVar:ETBAddCounter:Event$ Moved | Origin$ Stack | Destination$ Battlefield | ValidCard$ Card.IsRemembered | ReplaceWith$ ETBAddExtraCounter
 SVar:ETBAddExtraCounter:DB$ PutCounter | ETB$ True | Defined$ ReplacedCard | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ MoveToBattlefield
-SVar:MoveToBattlefield:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Defined$ ReplacedCard | SubAbility$ TrigRemoveSelf
-SVar:TrigRemoveSelf:DB$ ChangeZone | Defined$ Self | Origin$ Command | Destination$ Exile
+SVar:MoveToBattlefield:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Defined$ ReplacedCard | ReplacementResult$ Updated
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt:13]: Uuh no not like that, check out how others cards use it

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt]: If the result gets updated you don't need to handle the ETB anymore

File: forge-gui/res/cardsfolder/upcoming/hurkyls_prodigy.txt [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Name:Hurkyl's Prodigy
+ManaCost:2 U
+Types:Creature Human Soldier
+PT:1/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a tapped Powerstone token. (It's an artifact with "{T}: Add {C}. This mana can't be spent to cast a nonartifact spell.")
+SVar:TrigToken:DB$ Token | TokenTapped$ True | TokenScript$ c_a_powerstone
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigUnblockable | TriggerDescription$ Whenever CARDNAME attacks, you may pay {3}. If you go, CARDNAME can't be blocked this turn and it perpetually gets +2/+0.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/hurkyls_prodigy.txt:7]: If you do,

File: forge-gui/res/cardsfolder/upcoming/hurkyls_prodigy.txt [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+SVar:TrigToken:DB$ Token | TokenTapped$ True | TokenScript$ c_a_powerstone
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigUnblockable | TriggerDescription$ Whenever CARDNAME attacks, you may pay {3}. If you go, CARDNAME can't be blocked this turn and it perpetually gets +2/+0. 
+SVar:TrigUnblockable:AB$ Effect | Cost$ 3 | StaticAbilities$ Unblockable | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualPump | Duration$ Permanent | Name$ Hurkyl's Prodigy's Perpetual Effect
+SVar:Unblockable:Mode$ CantBlockBy | ValidAttacker$ Creature.Self | Description$ This creature can't be blocked.
+SVar:PerpetualPump:Mode$ Continuous | Affected$ Card.EffectSource | AddPower$ 2 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ EFFECTSOURCE perpetually gets +2/+0.
+SVar:HasAttackEffect:TRUE
+DeckHas:Ability$Token & Type$Artifact|Powerstone
+Oracle:When Hurkyl's Prodigy enter the battlefield, create a tapped Powerstone token.\nWhenever Hurkyl's Prodigy attacks, you may pay {3}. If you go, Hurkyl's Prodigy can't be blocked this turn and it perpetually gets +2/+0.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/hurkyls_prodigy.txt:14]: If you do,

File: forge-gui/res/cardsfolder/upcoming/norns_disassembly.txt [hunk 8]
```diff
@@ -0,0 +1,7 @@
+Name:Norn's Disassembly
+ManaCost:W
+Types:Enchantment
+A:AB$ ChangeZone | Cost$ Sac<1/Permanent.Historic/Historic permanent> | Origin$ Library | Destination$ Hand | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | ChangeType$ Card.Historic | ChangeNum$ 1 | SpellDescription$ Seek a historic card.
+DeckHas:Ability$Sacrifice
+DeckNeeds:Type$Legendary|Artifact|Planeswalker
+Oracle:{W}, Sacrifice a historic permanent: Seek a historic card.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/norns_disassembly.txt:7]: The ability costs 1W

**Review by paulsnoops — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/legion_of_clay.txt [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddKeyword$ Trample | Description$ Creatures you control have trample.
+T:Mode$ ChangesZoneAll | ValidCards$ Artifact.YouCtrl+nonToken | Origin$ Any | Destination$ Battlefield | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ Whenever one or more nontoken artifacts enter the battlefield under your control, creatures you control perpetually get +1/+1
+SVar:TrigEffect:DB$ Effect | RememberObjects$ Valid Creature.YouCtrl | StaticAbilities$ PerpetualP1P1 | Duration$ Permanent | SubAbility$ DBCleanup | Name$ Legion of Clay's Perpetual Effect
+SVar:PerpetualP1P1:Mode$ Continuous | Affected$ Card.IsRemembered | AddPower$ 1 | AddToughness$ 1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ Creatures you control perpetually get +1/+1.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHints:Type$Artifact|Golem|Construct|Thopter|Assembly-Worker|Myr|Necron|Juggernaut|Scarecrow
+Oracle:Creatures you control have trample.\nWhenever one or more nontoken artifacts enter the battlefield under your control, creatures you control perpetually get +1/+1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/legion_of_clay.txt:10]: Missing final .

File: forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt [hunk 10]
```diff
@@ -0,0 +1,15 @@
+Name:Patchplate Resolute
+ManaCost:3
+Types:Artifact Creature Soldier
+PT:3/3
+K:Unearth:1 W
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigBoon | TriggerDescription$ When CARDNAME enters or leaves the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it."
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Any | Execute$ TrigBoon | Secondary$ True | TriggerDescription$ When CARDNAME enters or leaves the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it."
+SVar:TrigBoon:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ SpellCast
+SVar:SpellCast:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | OneOff$ True | ActivationLimit$ 1 | TriggerZones$ Command | Execute$ ReplEffAddCounter | TriggerDescription$ When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt:9]: Try without ActivationLimit, imo the OneOff should already get rid of it anyway?

File: forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+SVar:ReplEffAddCounter:DB$ Effect | ReplacementEffects$ ETBAddCounter | RememberObjects$ TriggeredCard
+SVar:ETBAddCounter:Event$ Moved | Origin$ Stack | Destination$ Battlefield | ValidCard$ Card.IsRemembered | ReplaceWith$ ETBAddExtraCounter | ReplacementResult$ Updated
+SVar:ETBAddExtraCounter:DB$ PutCounter | ETB$ True | Defined$ ReplacedCard | CounterType$ P1P1 | CounterNum$ 1
+DeckHas:Ability$Counters
+DeckHints:Color$White
+Oracle:When Patchplate Resolute enters or leaves the battlefield, you get a boon with "When you cast your next creature spell, that creature enters the battlefield with an additional +1/+1 counter on it.\nUnearth {1}{W}
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/patchplate_resolute.txt:15]: Missing closing "

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**